core-services/prow/02_config/_plugins.yaml

```diff
@@ -141,11 +141,7 @@ bugzilla:
           branches:
             '*':
               add_external_link: true
-              dependent_bug_states:
-              - status: VERIFIED
-              - status: RELEASE_PENDING
-              - resolution: ERRATA
-                status: CLOSED
+              exclude_defaults: true
```

Comment:
Using `exclude_defaults` instead of `excludeDefaults` as in kubernetes/test-infra#15387. Added a [review][1] about it.

[1]: https://github.com/kubernetes/test-infra/pull/15387/files#r350581291